config/routes.rb

```diff
@@ -0,0 +1,8 @@
+# config/routes.rb
+Rails.application.routes.draw do
+  root 'greetings#random_greeting'
```

Comment:
- [ ] @mumbereh Great work generating a random greeting, however you are suppose to use the react app to display the random greeting, in your case you have used the rails greeting controller to display the greeting, kindly consider displaying the greeting using the react app, this is part of the project requirement.